bootstrap/kubeadm/types/v1beta1/bootstraptokenstring_test.go

```diff
@@ -227,23 +204,8 @@ func TestNewBootstrapTokenStringFromIDAndSecret(t *testing.T) {
 	for _, rt := range tests {
 		t.Run(rt.id, func(t *testing.T) {
 			actual, err := NewBootstrapTokenStringFromIDAndSecret(rt.id, rt.secret)
-			if (err != nil) != rt.expectedError {
-				t.Errorf(
-					"failed NewBootstrapTokenStringFromIDAndSecret for the token with id %q and secret %q\n\texpected error: %t\n\t  actual error: %v",
-					rt.id,
-					rt.secret,
-					rt.expectedError,
-					err,
-				)
-			} else if !reflect.DeepEqual(actual, rt.bts) {
-				t.Errorf(
-					"failed NewBootstrapTokenStringFromIDAndSecret for the token with id %q and secret %q\n\texpected: %v\n\t  actual: %v",
-					rt.id,
-					rt.secret,
-					rt.bts,
-					actual,
-				)
-			}
+			g.Expect(err != nil).To(Equal(rt.expectedError))
```

Comment:
I think we can be a bit clearer with a bit of extra verbosity here:
```suggestion
			if rt.expectedError {
				g.Expect(err).To(HaveOccurred())
			} else {
				g.Expect(err).NotTo(HaveOccurred())
			}
```

Reply:
done